omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -724,9 +731,40 @@
                     jq().toastmessage('showErrorToast', " Patient Not Stable " + instabilityCause + ". \n Only Stable Patients can be enrolled in " + jq("#165143").find("select").find(":selected").text());
                     return false;
                 }
+
+                if (getValue('art-regimen.value') != "" &amp;&amp; getValue('no-of-art-pills.value') == "") {
+                     getField('no-of-art-pills.error').html('No. of pills required').show();
```

Comment:
@dbaluku Error message - No of art-regimen pills required. Put the selected ART regimen as part of the error message

Reply:
Done